nml/actions/action4.py

```diff
@@ -82,7 +82,11 @@ def skip_action9(self):
     0xD0: {"random_id": True, "total": 0x400, "ids": list(range(0xD3FF, 0xCFFF, -1))},
     # Misc. persistent text ids, used to set properties.
     # Use Ids DC00..DCFF first to keep compatibility with older versions of OTTD.
-    0xDC: {"random_id": True, "total": 0x800, "ids": list(range(0xDBFF, 0xD7FF, -1)) + list(range(0xDFFF, 0xDBFF, -1))},
+    0xDC: {
+        "random_id": True,
+        "total": 0x2800,
+        "ids": list(range(0xFFFF, 0xDFFF, -1)) + list(range(0xDBFF, 0xD7FF, -1)) + list(range(0xDFFF, 0xDBFF, -1)),
```

Comment:
This contradicts the comment from line 84. Now 0xFFFF is the first string ID to be used, raising the OpenTTD requirements.

Reply:
Actually the values are poped from the list, so `DCFF..DC00` must be at the very end, and that's done with the last range.